src/main/java/duke/ExitCommand.java

```diff
@@ -0,0 +1,14 @@
+package duke;
+
```

Comment:
Perhaps you can have a Javadoc comment here briefly describing the class?

Reply:
Alright thanks